packages/node_modules/@ciscospark/internal-plugin-presence/src/index.js

```diff
@@ -0,0 +1,85 @@
+/**!
+ *
+ * Copyright (c) 2015-2017 Cisco Systems, Inc. See LICENSE file.
+ * @private
+ */
+import '@ciscospark/internal-plugin-wdm';
+
+import {registerInternalPlugin} from '@ciscospark/spark-core';
+import {has} from 'lodash';
+
+import Presence from './presence';
+import config from './config';
+
+registerInternalPlugin(`presence`, Presence, {
+  payloadTransformer: {
+    predicates: [
+      {
+        name: `normalizeSingleStatusResponse`,
+        direction: `inbound`,
+        test(ctx, response) {
+          return Promise.resolve(has(response, `body.eventType`));
```

Comment:
all traffic. the idea was to let us add new apis that would *just work* without needing to make sdk changes.